examples/web/src/features/lambda/browser/styles.css

```
@@ -1,13 +1,17 @@
+.lambda-surface {
     * {
       box-sizing: border-box;
       margin: 0;
       padding: 0;
     }
 
-    body {
+    & {
+      box-sizing: border-box;
+      margin: 0;
       font-family: 'Consolas', 'Monaco', 'Courier New', monospace;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Consolas" (font-family-name-quotes)

(font-family-name-quotes)